crowdloans/rewards/polkadot.json

```diff
@@ -1,4 +1,83 @@
 {
+  "2019": {
+    "cDot": "${amount * 1}",
+    "reward": {
+      "total": "5% of Total Supply",
+      "fixed": [{ "value": 0.48, "suffix": "+", "unit": "LAYR" }],
+      "unit": "LAYR",
+      "detail": [
+        {
+          "label": "Est. Project Rewards",
+          "formula": "${amount * 0.48}"
+        },
+        {
+          "label": "Early Bird Bonus",
+          "formula": [
+            {
+              "condition": { "rangeDate": ["12/15/21", "12/15/21"] },
+              "value": "${amount * .48 * 0.15}"
```

Comment:
the bonus is 5% for first 24 hrs in notion doc https://www.notion.so/parallelfinance/Batch-2-DOT-Crowdloan-345f2f01d71643febd8b3c6c935615eb